.github/CODEOWNERS

```diff
@@ -1,4 +1 @@
-# CODEOWNERS: https://help.github.com/articles/about-codeowners/
-
-* @chadoh
-* @vgrichina
+.github/workflows/ @morgsmccauley @andy-haynes
```

Comment:
I think we need more than 2 people here. Maybe you can add @MaximusHaximus ? I can also volunteer if you're willing to do some knowledge transfer. Also, people in the git history of these files that are still in the org may be useful.